src/main/java/com/faforever/client/vault/replay/ReplayVaultController.java

```diff
@@ -120,6 +124,25 @@ protected void loadLocalReplaysInBackground() {
   @NotNull
   private TableRow<Replay> replayRowFactory() {
     TableRow<Replay> row = new TableRow<>();
+    ContextMenu contextMenu = new ContextMenu();
+    MenuItem copyReplayId = new MenuItem("Copy Replay Id");
```

Comment:
we use internationalization so this needs to be translatable. Use the i18n service. Just make a constructor parameter like all the other services. You then enter the english translation in the message.properties file in the resources/i18n folder

Reply:
what do I do with the other languages?

Reply:
Nothing they are translated on poeditor.com